packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -674,22 +674,32 @@ export class RunningSummarizer implements IDisposable {
 						});
 						await delay(delaySeconds * 1000);
 					}
-				}
 
-				if (this.shouldAbortOnSummaryFailure) {
-					this.mc.logger.sendTelemetryEvent(
-						{
-							eventName: "ClosingSummarizerOnSummaryStale",
-							reason,
-							message: lastResult?.message,
-						},
-						lastResult?.error,
-					);
+					if (this.shouldAbortOnSummaryFailure) {
+						// There's no need to delay here as we already have summarizeAttemptDelay given to us by the server.
+						if (delaySeconds === 0) {
+							const closeSummarizerDelayOverrideMs =
+								this.mc.config.getNumber(
```

Comment:
Can you please add all the FGs we are using / adding to "ContainerLoadStats" event in containerRuntime.ts? Thanks!